.gitignore

```diff
@@ -25,7 +25,6 @@ yarn-error.log*
 .pnpm-debug.log*
 
 # local env files
-.env
```

Comment:
why?

Reply:
Cause the pagedir example is running on port 3010 and to get it to work we must override the UPLOADTHING_URL and i thought it could be commited? It's also leftovers from when i tested out the 100s of errors during the example builds though so can revert it in here